README.md

```diff
@@ -22,7 +23,7 @@ To build the project:
 
 ```
 cd hps-mc; mkdir build; cd build
-cmake -DCMAKE_INSTALL_PREFIX=../install ..
+cmake -DCMAKE_INSTALL_PREFIX=../install -DGSL_ROOT_DIR=$GSL_ROOT_DIR ..
```

Comment:
Please include a line somewhere as an example of where you set this. Is it just the directory you get from unzipping the tarball?

Reply:
I updated the README just now. This is just the installation prefix of GSL (from the `--prefix` option when it was installed).

Reply:
thx